pages/submissions/[id]/index.tsx

```diff
@@ -104,15 +104,6 @@ export const getServerSideProps: GetServerSideProps = async ({ params }) => {
       },
     };
 
-  if (data.submissionState !== 'In progress') {
```

Comment:
where did this go?

Reply:
I didn't do that.

Reply:
let's bring it back then - can you make this file the way it was pre-changes?

Reply:
ok